packages/multi-test/src/wasm.rs

```diff
@@ -848,23 +833,23 @@ where
 }
 
 // this parses the result from a wasm contract init
-pub fn parse_contract_addr(data: &Option<Binary>) -> Result<Addr, String> {
+pub fn parse_contract_addr(data: &Option<Binary>) -> Result<Addr> {
     let bin = data
         .as_ref()
-        .ok_or_else(|| "No data response".to_string())?
+        .ok_or_else(|| anyhow!("No data response"))?
```

Comment:
I see, `anyhow!` is the constructor helper macro, and `bail!` is the return helper macro.

Reply:
`anyhow!` is more than constructor, it is macro creating wrapper for an error value (same for `bail!`). It basically allows to create `anyhow::Error` from types which are not `Into<anyhow::Error>` (maybe they are not `Send`, or not `std::Error`). It only requires internals to be `Display + Debug`, which string hopefully is. And also those have neat functionality of standard string formatting.